### PR TITLE
[refactor-WIP](TaskWorkerPool) add CreateTableTaskPool class for CREATE_TABLE task

### PR DESCRIPTION
**Pull request by bobhan1**
# Proposed changes

Issue Number: #19575 

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [x] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by bobhan1** [comment]
buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by bobhan1** [comment]
run buildall

File: be/src/agent/task_worker_pool.h [hunk 1]
```diff
@@ -263,4 +267,14 @@ class TaskWorkerPool {
 
     DISALLOW_COPY_AND_ASSIGN(TaskWorkerPool);
 }; // class TaskWorkerPool
+
+class CreateTableTaskPool : public TaskWorkerPool {
+public:
+    CreateTableTaskPool(ExecEnv* env, const TMasterInfo& master_info, ThreadModel thread_model);
```
Comment (yiguolei) [inline be/src/agent/task_worker_pool.h:273]: 这里不要传递master info了，这个会变，直接每次都从exec env 里拿就好了。 （虽然你这里用了引用，但是未来可能忘记，容易出错）

**Comment by bobhan1** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**